--- usr/lib/ruby/vendor_ruby/puppet/provider/service/debian.rb~	2014-01-06 22:36:36.000000000 +0000
+++ usr/lib/ruby/vendor_ruby/puppet/provider/service/debian.rb	2014-02-12 08:37:30.659576483 +0000
@@ -36,14 +36,12 @@
     # 104 is the exit status when you query start an enabled service.
     # 106 is the exit status when the policy layer supplies a fallback action
     # See x-man-page://invoke-rc.d
-    if [104, 106].include?($CHILD_STATUS.exitstatus)
-      return :true
-    elsif [105].include?($CHILD_STATUS.exitstatus)
+    if [104, 105, 106].include?($CHILD_STATUS.exitstatus)
       # 105 is unknown, which generally means the the iniscript does not support query
       # The debian policy states that the initscript should support methods of query
-      # For those that do not, peform the checks manually
+      # For those that do not, perform the checks manually
       # http://www.debian.org/doc/debian-policy/ch-opersys.html
-      if get_start_link_count >= 4
+      if get_start_link_count >= 1
         return :true
       else
         return :false
