--- backend.rb  2014-10-08 16:43:30.204845671 +0000
+++ backend.rb2 2014-10-08 16:44:10.168149074 +0000
@@ -227,6 +227,13 @@
         answer = parse_string(default, scope) if answer.nil? and default.is_a?(String)
 
         return default if answer.nil?
+        if answer.is_a?(String) and (answer.start_with?('[') or answer.start_with?('{'))
+          proper_answer = eval(answer)
+        elsif answer.is_a?(String) and (answer.eql?('true') or answer.eql?('false'))
+          proper_answer = eval(answer)
+        else
+          proper_answer = answer
+        end
         return proper_answer
       end
